srv/salt/ceph/rgw/users/users.d/README

```diff
@@ -0,0 +1,8 @@
+Initial Users
+
+The /srv/salt/ceph/rgw/files/users.j2 will include any rgw users created in
+this directory.  Add users to one or more files with a yml extension.  Each entry is indented six spaces to form a proper dictionary.
```

Comment:
I believe we are shooting ourselves in the foot with this: `Each entry is indented six spaces to form a proper dictionary.` 
It's so easy to miss this and by mistake get the wrong number of indentation spaces.

I believe the user should just declare a list of users with zero spaces of indentation, and in `/srv/salt/ceph/rgw/files/users.j2` we process this list in jinja and add the indentation spaces.
What do you think?

Reply:
Agree, this would be something that would end up being set wrong, and hard to track down later

Reply:
Will slsutil.renderer process it correctly?  I am not reading the file.  I am relying on Salt.

As far as tracking it down, we can have the module complain verify the yaml.  We do that during Stage 2 and it's not foolproof either.

Technically, this is no different than what was in the pillar.  Users can get the structure wrong there as well.

Reply:
The idea is to instead of just include the user's file as in [srv/salt/ceph/rgw/files/users.j2:10](https://github.com/SUSE/DeepSea/pull/607/files#diff-c62df72fdd6c23b5f05fe33ba66bdcebR10),
process the file contents it a for loop and correctly create the indentation steps.

Something like this (not sure if it works, didn't test it):

```
{% for user in salt['sls.renderer'](file) %}
      - { uid: "{{ user['uid'] }}", name: "{{ user['name'] }}", email: "{{ user['email'] }}" }
{% endfor %}
```

Reply:
More fields are allowed than just those.  And yaml supports more than one structure, so I wasn't trying to suggest/enforce the one liner per entry.

The double for loop could be an issue, but I can see if something silly will work (i.e. indent the include directive).